ui/client/index.js

```diff
@@ -78,10 +78,31 @@ export default function Main() {
   );
 }
 
+const changeFavicon = (faviconFileName) => {
+  const link = document.querySelector("link[rel~='icon']");
+  if (link) {
+    link.href = `/assets/${faviconFileName}`;
+  } else {
+    const newLink = document.createElement('link');
+    newLink.rel = 'icon';
+    newLink.href = `/assets/${faviconFileName}`;
+    document.head.appendChild(newLink);
+  }
+};
+
+if (process.env.COMPANY_BRANDING === 'ifpri') {
```

Comment:
I played around with changing this to use the functions in Branding.js, but it ended up making it harder to read and didn't reduce much code, so I kept it as is.